tests/components/unifi/test_switch.py

```diff
@@ -367,8 +367,8 @@
     await hass.services.async_call(
         "switch", "turn_on", {"entity_id": "switch.poe_client_1"}, blocking=True
     )
-    assert len(controller.mock_requests) == 5
-    assert controller.mock_requests[3] == {
+    assert len(controller.mock_requests) == 6
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5028567)